docs/VnVPlan/VnVPlan.tex

```diff
@@ -900,18 +1045,18 @@ \subsection{Traceability Between Test Cases and Requirements}
   UH-UP1 & \\ \hline
   UH-UP2 & \\ \hline
   UH-AS1 & \\ \hline
-  PR-SL1 & \\ \hline
+  PR-SL1 & TC-PR-1\\ \hline
   PR-SL2 & \\ \hline
   PR-SL3 & \\ \hline
-  PR-SC1 & \\ \hline
+  PR-SC1 & TC-PR-2\\ \hline
   PR-SC2 & \\ \hline
-  PR-PA1 & \\ \hline
+  PR-PA1 & TC-PR-3\\ \hline
   PR-PA2 & \\ \hline
-  PR-RFT1 & \\ \hline
-  PR-RFT2 & \\ \hline
-  PR-CR1 & \\ \hline
-  PR-CR2 & \\ \hline
-  PR-SE1 & \\ \hline
+  PR-RFT1 & TC-PR-4\\ \hline
+  PR-RFT2 & TC-PR-5\\ \hline
+  PR-CR1 & TC-PR-6\\ \hline
+  PR-CR2 & TC-PR-7\\ \hline
+  PR-SE1 & TC-PR-8\\ \hline
   PR-SE2 & \\ \hline
   PR-LR1 & \\ \hline
```

Comment:
this is a bit difficult to verify, but I guess we can just include a vague thing explaining how development team will evaluate the modularity of the system

Reply:
can say things such as ensuring testability (i.e. actual unit can be tested and the system isn't tightly coupled) and ensuring changing x part of the system doesn't affect y